### PR TITLE
Move `update_codelists` to `codelists update`

### PR DESCRIPTION
**Pull request by evansd**
This is in preparation for adding more codelist related commands.

Technically this is a major version bump but as no one is officially
using this tool yet we can stick with 1.x for now.

Also script up the dependency updating process and run it.

### CODE REVIEW

File: opensafely/_vendor/certifi/cacert.pem [hunk 1]
```diff
@@ -4604,3 +4282,44 @@ AZKXRRJ+oPM+rRk6ct30UJMDEr5E0k9BpIycnR+j9sKS50gU/k6bpZFXrsY3crsC
 MGclCrEMXu6pY5Jv5ZAL/mYiykf9ijH3g/56vxC+GCsej/YpHpRZ744hN8tRmKVu
 Sw==
 -----END CERTIFICATE-----
+
```
Comment (CarolineMorton) [inline opensafely/_vendor/certifi/cacert.pem:4285]: Should this certificate be visible or can it be stashed within the repo settings as TOKEN. I've not done this with github actions but have with Travis and it seemed ok?

**Review by CarolineMorton — APPROVED** [review]
Looks fine - not sure if we want the `.pem` published?

File: opensafely/_vendor/certifi/cacert.pem (hunk 1 above)
Comment (evansd) [reply opensafely/_vendor/certifi/cacert.pem:4285]: Yes, we do need this. There are public root certificates which the `requests` library uses for verifying HTTPS connections. As we're vendoring in all our dependencies these get pulled in as well.

Always good to query anything that looks like a key though!

Comment (CarolineMorton) [reply opensafely/_vendor/certifi/cacert.pem:4285]: cool thanks for the explanation

**Pull request merged**